### ISSUE TITLE
Fails to compile on current nightly, after target_feature stabilisation

### ISSUE THREAD

**Issue by g2p** [nagisa/rust_rdrand#3]
```
rustup show
…
active toolchain
----------------

nightly-x86_64-unknown-linux-gnu (directory override for '/home/g2p/src/rust_rdrand')
rustc 1.28.0-nightly (990d8aa74 2018-05-25)
```

target_feature seems to have changed while being stabilised

```
warning: this feature has been stable since 1.27.0. Attribute no longer needed
  --> src/lib.rs:15:12
   |
15 | #![feature(target_feature, asm, platform_intrinsics)]
   |            ^^^^^^^^^^^^^^
   |
   = note: #[warn(stable_features)] on by default

error: #[target_feature(..)] can only be applied to `unsafe` function
  --> src/lib.rs:96:5
   |
96 |     #[target_feature="+rdrnd"]
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^

error: #[target_feature] attribute must be of the form #[target_feature(..)]
  --> src/lib.rs:96:5
   |
96 |     #[target_feature="+rdrnd"]
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^
```

**Comment by nagisa** [comment]
I have an in-progress rewrite to migrate rust_rdrand to rand-core and coresimd. The migration was blocked on `[T]::align_to`, but since that has landed recently, I’ll be able to finish the migration.

**Comment by g2p** [comment]
I have minimal changes to make it compile on stable Rust, if you're interested. Should I send the PR?

**Comment by nagisa** [comment]
Sure.
